File: 은지/정수 내림차순으로 배치하기.py
def solution(n):
    # 정수 => 문자열 배열
    arr = []
    for x in str(n):
        arr.append(x)
    # 내림차순 정렬
    arr.sort(reverse=True)
    # 배열 => 문자열 => 정수
    answer = int("".join(arr))
    return answer

File: 준영/프로그래머스/LV1/정수 내림차순으로 배치하기.js
function solution(n) {
  var answer = 0
  var arr = []
  for (const i of String(n)) {
    arr.push(i)
  }
  arr.sort((a, b) => b - a)
  answer = arr.join('')
  answer = parseInt(answer)
  return answer
}

solution(118372)